src/components/FilterList.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const DefaultItem = ({ serviceName, departmentName }) => (
+  <div>
+    <h3>{serviceName}</h3>
+    <p>{departmentName}</p>
+  </div>
+);
+
+const FilterList = ({
```

Comment:
We are going to want to add proptypes for this. I'm thinking that we could require the 'renderItem` function as required and remove the default component.

My thinking  is that, in every use case we will not have standard layout. I jsut don't see us using the default? If we do, we should probably change the props to `title` and `description` to be more generic. @tmccoy529 @sgrg1 what do you think?

Reply:
removed default

Reply:
please add the prop types.

Reply:
done